scheduler/src/cook/mesos/schema.clj

```diff
@@ -186,6 +186,12 @@ for a job. E.g. {:resources {:cpus 4 :mem 3} :constraints {\"unique_host_constra
     :db/valueType :db.type/ref
     :db/cardinality :db.cardinality/one
     :db.install/_attribute :db.part/db}
+   {:db/id (d/tempid :db.part/db)
+    :db/ident :job/supports-data-locality
```

Comment:
"Supports" seems a little bit odd. Maybe "participates in" (as the `:db/doc` states)? Or maybe we just shorten to `:job/data-local`?

Reply:
Done